python/foxglove-sdk/pyproject.toml

```diff
@@ -51,4 +51,4 @@ warn_unreachable = true
 
 [tool.black]
 include = '\.pyi?$'
-extend-exclude = "python/foxglove/_protobuf"
+extend-exclude = "python/foxglove/(_protobuf|_foxglove_py/schemas.pyi)"
```

Comment:
we could run `black` after generating code

Reply:
👍 The typescript script now runs `black` and `rustfmt`. If either tool isn't installed, this step is skipped.

Reply:
can you make sure the format step always runs in CI (e.g. fails if either tool is not installed), so that we don't mark the PR as green unless the code is correctly formatted.

e.g. from this step https://github.com/foxglove/foxglove-sdk/blob/main/.github/workflows/ci.yml#L76-L89